### PR TITLE
Updated exoplayer version from 2.15.1 to r.2.16.1

### PR DESCRIPTION
**Pull request by BFK78**
## Link to pull request in Documentation repository

Documentation: home-assistant/companion.home-assistant#

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @BFK78, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;android#2144).

Once you do that we will be able to review and accept this pull request.

Thanks!

File: app/build.gradle.kts [hunk 1]
```diff
@@ -180,9 +180,9 @@ dependencies {
     implementation("androidx.biometric:biometric:1.1.0")
     implementation("androidx.webkit:webkit:1.4.0")
 
-    implementation("com.google.android.exoplayer:exoplayer-core:2.15.1")
-    implementation("com.google.android.exoplayer:exoplayer-hls:2.15.1")
-    implementation("com.google.android.exoplayer:exoplayer-ui:2.15.1")
+    implementation("com.google.android.exoplayer:exoplayer-core:r2.16.1")
```
Comment (dshokouhi) [inline app/build.gradle.kts:183]: Not sure if the `r` should be there. According to Android Studio its just the number that is changing. Also `cronet` has an update as well. I also see some deprecations in the changelog, have you tested to make sure its working properly? 

image

**Review by dshokouhi — CHANGES_REQUESTED** [review]
In order to continue with this PR it now needs to be rebased and updated to match the new format

**Comment by JBassett** [comment]
Upgraded in https://github.com/home-assistant/android/pull/2556

**Pull request closed without merging**